README.md

```diff
@@ -47,7 +47,7 @@ installed on your cluster before you install Chains.
 To install the latest version of Chains to your Kubernetes cluster, run:
 
 ```shell
-kubectl apply --filename https://storage.googleapis.com/tekton-releases/chains/previous/v0.14.0/release.yaml
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/chains/previous/v0.15.0/release.yaml
```

Comment:
How about using https://storage.googleapis.com/tekton-releases/chains/latest/release.yaml so we don't have to worry about bumping this again?

Reply:
Yeah I think that makes sense